### ISSUE TITLE
Linux server does not start after new install

### ISSUE THREAD

**Issue by port13** [PocketMine/PocketMine-MP#2479]
Today I successfully downloaded the Linux version of PocketMine-MP server. However, nothing happens after running ./start.sh. If I run start.sh from the GUI, a terminal window flashes on screen and immediately disappears.

**Comment by sithsiri** [comment]
Try deleting and reinstalling?
Or you can just do it on Windows or Mac. Unless you cant. Sorry if I'm not very helpful.

**Comment by port13** [comment]
I tried deleting and reinstalling. No luck.
Following the Linux instructions from PocketMine's website did not work for me (the download does not start and no error is shown). I followed the Linux instructions from the Wiki. I did modify the download line by pointing it to the development version:

It was attempting to download the 32-bit PHP build PHP_5.6.4_x86_Linux.tar.gz and said invalid build detected. I manually downloaded the next latest 32-bit build directly from Jenkins which was build #129. After extracting those files, I get the same result when launching start.sh--the terminal window briefly flashes on scree.

**Comment by port13** [comment]
curl -sL http://cdn.pocketmine.net/installer.sh | bash -s - -v development

This was the download command I meant to include in the previous coment.

**Comment by shoghicp** [comment]
Hmm, we are updating the process and scripts used to download PM builds, starting with the development builds. Try downloading the beta build

**Comment by port13** [comment]
Downloading the beta build had the same result. Still getting the invalid build detected on Linux 32-bit PHP build PHP_5.6.4_x86_Linux.tar.gz

**Comment by shoghicp** [comment]
@port13 Are you downloading for MacOS or Linux? Try using wget as the homepage tells you to do.
If the build is invalid, the compiler will try to get you a working binary on your same system.

**Comment by port13** [comment]
I'm downloading for Linux. Yesterday when I tried the wget per the homepage nothing happened. No error message but the download didn't start. That's when I went with the curl command from the Wiki instructions which was able to complete the download except for the PHP issue mentioned above. Today when I tried the wget command per the homepage, the terminal appears to hang. The download doesn't start but I don't get a command prompt.
